src/screens/Home/components/SuggestedPodcastCarousel/SuggestedPodcastCarousel.tsx

```diff
@@ -19,18 +19,27 @@ const SuggestedPodcastCarousel: React.FC<Props> = ({ data, screenPadding }) => {
   const navigation = useNavigation<PodcastScreenNavigationProp>()
 
   const renderCarouselItem = ({
-    item: { name, user, image, id }
+    item: { name, user, image, id, episodes }
   }: CarouselItem) => {
     const handleNavigateToPodcast = () => {
       navigation.navigate(NavigationScreen.PODCAST, { id })
     }
 
+    const handleNavigateToEpisode = () => {
+      navigation.navigate(NavigationScreen.EPISODE, {
+        id: episodes[0].id,
```

Comment:
What does it mean `0`?
Let's use [destructuring assignment](https://javascript.info/destructuring-assignment): `const [firstEpisode] = episodes`